includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -696,6 +699,11 @@ private function getCommunity() {
 		return $data;
 	}
 
+	private function isLocalSitemapAvailable() {
+		$default = true; // $wgEnableLocalSitemapPageExt = true; in CommonSettings
```

Comment:
It's taken a line below. This is just the default value if none is taken. https://github.com/Wikia/app/blob/dev/extensions/wikia/WikiFactory/WikiFactory.php#L1010